### ISSUE TITLE
Version 0.148.6 missing

### ISSUE THREAD

**Issue by matukaa** [sethmachine/valheim-server-docker#11]
Hey!

I am not entirely sure if there is something I do wrong, but I am unable to update to 0.148.6.
The :latest tag on docker seems to be built 4 days ago which is prior to the new version.
I tried manually rebuilding the image after pulling the repo (even with --no-cache), yet somehow steamcmd only provides version 0.147.3. Is this an issue on their side?

**Comment by gerardq675** [comment]
Hi Matukaa,

I don't work on this repo but I was able to get it working on my server. I realised that the instructions are potentially slightly incorrect. Use:
`docker build -t sethmachine/valheim-server --no-cache .`

And not

`docker build -t sethmachineio/valheim-server --no-cache .`

That's listed. Hopefully this works for you!

Obviously when you go to host the image make sure it says sethmachine/valheim-server and not sethmachineio/valheim-server

**Comment by matukaa** [comment]
I have downloaded the Dockerfile and the start server script, and made my own build (named it macmini/valheim-server). I used --no-cache and I am quite sure the docker run command is also correct, however when I boot up my client and try to join, server shows up with version 0.147.3. I have also tried pulling the latest steamcmd image to make sure it contains the updated version for the server, but nothing else changes
![Untitled](https://user-images.githubusercontent.com/7106864/112458756-cddda380-8d65-11eb-8604-5cf3949c046f.png)

**Comment by matukaa** [comment]
I am entirely unsure why, but I stopped the server again, deleted container, re-run the build and run commands, and now it shows up as 0.148.6. I have used the same set of commands (build with --no-cache and run) like 3 times now (using up arrows, so I didn't typo) and now it suddenly worked.
My apologies for the issue, I will be closing it now, must've been something that I did wrong, I am kinda new to the whole docker scene.

Cheers and thanks
